eos-split-flatpak-repo

```diff
@@ -205,6 +242,7 @@ def gather_refs(repo, os_remotes, flatpak_remotes):
     flatpak_refs = []
     other_refs = []
     _, all_refs = repo.list_refs_ext(None, OSTree.RepoListRefsExtFlags.NONE)
+    stats.num_refs = len(all_refs)
```

Comment:
Perhaps we can break this down and report the number of OS refs, flatpak refs, and other refs?

If we were feeling fancy we could also report the number of Flatpak apps installed – but I suppose you can't just assume that flatpak refs = 2 * flatpak apps (normal ref and deploy/ ref) because apps can have 1 or more extensions.

Reply:
Sure, we could do that. I can also just open the `Flatpak.Installation` on get the info as interpreted by flatpak itself. That would also allow us to see the number of pulled but not installed refs.